v2/pkg/controller/mpi_job_controller.go

```diff
@@ -918,38 +909,42 @@ func (c *MPIJobController) deleteWorkerPods(mpiJob *kubeflow.MPIJob) error {
 	return nil
 }
 
-func (c *MPIJobController) updateMPIJobStatus(mpiJob *kubeflow.MPIJob, launcher *corev1.Pod, worker []*corev1.Pod) error {
+func (c *MPIJobController) updateMPIJobStatus(mpiJob *kubeflow.MPIJob, launcher *batchv1.Job, worker []*corev1.Pod) error {
 	oldStatus := mpiJob.Status.DeepCopy()
+	// Job.status.Active accounts for Pending and Running pods. Querying
+	// the pods to get only the Running pods.
+	launcherRunning, err := c.jobRunningPods(launcher)
```

Comment:
Looking at this line, it wasn't very clear what "jobRunningPods" does - maybe something like this would be more readable?

launcherPodsCnt, err := c.runningPodsCnt(launcher)

Reply:
changed to `jobRunningPodsCount`